node/src/android/.idea/modules.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Most of the files under `.idea` should be under source control, including this one. [See StackOverflow discussion.](http://stackoverflow.com/questions/16736856/what-should-be-in-my-gitignore-for-an-android-studio-project)